mapdata/WarcraftLegacies/AbilityData/Skin/925910093.json

```
@@ -0,0 +1,86 @@
+{
+  "OldId": 1986086977,
+  "NewId": 925910093,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(Yellow-Muradin)"
```

```suggestion
      "Value": "(Muradin)"
```
Might as well clean this one up